### ISSUE TITLE
Tab Content Focus

### ISSUE THREAD

**Issue by ccamarat** [bootstrap-vue/bootstrap-vue#2780]
Tab content is showing a blue border around it starting @rc12. Issue is visible on [docs site for tabs](https://bootstrap-vue.js.org/docs/components/tabs/); just open in Chrome and click the text "I'm the first fading tab":

![image](https://user-images.githubusercontent.com/1820184/53889624-91527a00-3fdb-11e9-968e-441d2afdb1d7.png)

It's worth mentioning that in past releases the focus ring would appear around the tabs themselves if the user clicked in the tab header (as per the W3C ARIA spec); that no longer appears to be the case.

**Comment by tmorehouse** [comment]
Tabs are going under a major re-write to handle/fix certain reactivity edge cases.

As per the ARIA spec, the content of the `tab-pane` should be focusable, to allow the keyboard only or screen reader user to jump to the content easily and have it read (since it is not an `aria-live` region)

https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html

![image](https://user-images.githubusercontent.com/2781561/53895261-fe7c0480-4007-11e9-87e3-197a95cfba3c.png)

**Comment by tmorehouse** [comment]
By setting the prop `no-key-nav`, the tabidex on teh tab content will be removed.

**Comment by ccamarat** [comment]
Thanks @tmorehouse. I notice that the focus behavior for the tabs themselves changed between rc11 and rc12; previously they had the focus outline and now its gone... I'm guessing this is on your overhaul list already! Thanks for all the great work you guys are doing!

**Comment by tmorehouse** [comment]
Yeah, we switched from a "widget" style nav tab buttons (where hte widget had a tabindex of 0), to the roving tabindex method (where the active tab-button had the tabindex of 0 and the non active buttons had a tabindex of -1).

Although this may go away in the revamped tabs (where basically we default back to regular tab order instead of keynav abilities). Due to keynav not working with extra content-less tab buttons.

**Comment by DM2489** [comment]
Surely the tabindex highlight should only be displayed when using the keyboard to tab to the content, and not via a mouse click?

**Comment by tmorehouse** [comment]
The highlight is needed for keyboard-only users in order to tell which tab/element is focused.

Unfortunately there is no "real" difference between mouse click and "tab into".  Many modern browsers use heuristics to determine if the focus ring should be shown, but they can't always detect.

You can always create a CSS style to override the `outline` that the browser generates (blue ring on chrome, dotted line on firefox).  But then you have partially broken accessibility for keyboard only users.

**Comment by DM2489** [comment]
I see - thanks for the explanation.
